src/tests/ftest/security/pool_connect_init.yaml

```diff
@@ -0,0 +1,26 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+ test_machines:
+  - boro-A
+timeout: 300
+server_config:
+   name: daos_server
+poolparams:
+   createmode:
+     mode: 511
+   createset:
+     setname: daos_server
+   createsize:
+       size: 107374182
+object_class: !mux
```

Comment:
Are any of these object class items needed? I don't see them referenced at all in the test above.

Reply:
poolparams:  createset:    setname: daos_server   and
                      createsize:   size:        107374182           are needed to create a pool.
as discussed, mode: 511 will be removed.
Thanks.

Reply:
I think David was talking about object class below which seems to be not used anywhere in the code. I think this can be removed if not used.

Reply:
Current coed will be failed without those parameters.
We have decided to add the parameters in yaml file for now